service/migration_manager.cc

```diff
@@ -638,19 +648,26 @@ future<> migration_manager::announce_column_family_update(schema_ptr cfm, bool f
 #endif
         mlogger.info("Update table '{}.{}' From {} To {}", cfm->ks_name(), cfm->cf_name(), *old_schema, *cfm);
         auto&& keyspace = db.find_keyspace(cfm->ks_name()).metadata();
-        auto mutations = db::schema_tables::make_update_table_mutations(keyspace, old_schema, cfm, ts, from_thrift);
-        return map_reduce(view_updates,
-            [keyspace, ts] (auto&& view) {
-                auto& old_view = keyspace->cf_meta_data().at(view->cf_name());
-                mlogger.info("Update view '{}.{}' From {} To {}", view->ks_name(), view->cf_name(), *old_view, *view);
-                auto mutations = db::schema_tables::make_update_view_mutations(keyspace, view_ptr(old_view), std::move(view), ts, false);
-                return make_ready_future<std::vector<mutation>>(std::move(mutations));
-            }, std::move(mutations),
-            [] (auto&& result, auto&& view_mutations) {
-                std::move(view_mutations.begin(), view_mutations.end(), std::back_inserter(result));
-                return std::move(result);
-            })
-        .then([keyspace, announce_locally] (auto&& mutations) {
+
+        return seastar::async([this, cfm, old_schema, ts, keyspace, from_thrift, view_updates] {
```

Comment:
Would have been better to add these with a half indent and reindent in a later patch.